src/components/Collapsible/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { ReactElement } from 'react';
+import clsx from 'clsx';
+import { Flipped, Flipper } from 'react-flip-toolkit';
+import { Props } from './types';
+
+
+/*
+ ** Collapsible component uses CSS transitions to smoothly hide and show a block of content.
+ */
+function Collapsible({
+    className,
+    anchorContent,
+    isCollapsed = true,
+    children,
+    ...otherProps
+}: Props): ReactElement {
+
+
+    const [collapsed, setCollapsed] = React.useState(isCollapsed);
+    const wrapperClasses = clsx(
+        'brew-Collapsible',
+        className,
+    );
+    const contentClasses = clsx(
```

Comment:
I've been adding 2 spaces pretty much before and after everything even if it's related like these constants. @lawren  thoughts?